drivers/pwm/Kconfig.nrfx

```diff
@@ -0,0 +1,326 @@
+# Copyright (c) 2017 - 2018, Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+menuconfig PWM_NRFX
+	bool "nRF PWM nrfx drivers"
+	depends on SOC_SERIES_NRF52X
+	help
+	  Enable support for nrfx Hardware PWM drivers for nRF52 MCU series.
+
+if PWM_NRFX
+
+config PWM_0_NRFX_PWM
+	bool "nRF HW PWM 0"
+	depends on SOC_SERIES_NRF52X
+	select NRFX_PWM
+	help
+	  Enable nRF Hardware PWM module 0.
+
+if PWM_0_NRFX_PWM
```

Comment:
Should PWM_0_NRFX_DEV_NAME then be PWM_0_DEV_NAME?

Reply:
Actually, `PWM_0_NAME`. To keep consistency with what we've got for SPI, I2C, or ADC.

And for other config entries here, I think we should use the convention `PWM_x_NRF_...`, since they are not strictly related to the nrfx driver but rather to the nRF peripherals themselves. We do the same in SPI, I2C (in #7788), or UART.